modules/dicom_archive/jsx/entrypoint.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import DicomArchive from './dicom_archive';
+
+/**
```

Comment:
If you want to have an entry point file for every module, I think it's sensible to name it `index.js` as it seem to be the convention in node modules